src/rmp/src/abc_library_factory.cpp

```diff
@@ -236,6 +238,13 @@ std::vector<abc::SC_Pin*> AbcLibraryFactory::CreateAbcOutputPins(
     std::unordered_set<std::string> pins;
     for (sta::TimingArcSet* arc_set :
          cell_port->libertyCell()->timingArcSets(nullptr, cell_port)) {
+      // If the from pin is an output it means that this is
+      // an output to output timing arc, and not something we
+      // care about or can represent in ABC. Skip it.
```

Comment:
What does it mean to drop such an arc?  Will abc be able to preserve the logical function without this?